src/components/file/examples/file-menu-items.tsx

```
@@ -0,0 +1,77 @@
+        },
+    ];
+
+    public render() {
+        return [
+            <limel-file
+                label="Attach a file"
+                value={this.value}
+                menuItems={this.menuItems}
+                onChange={this.handleChange}
+                onMenuItemSelected={this.handleMenuItemSelected}
+            />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)